Formula/r/ratchet.rb

```diff
@@ -1,8 +1,9 @@
 class Ratchet < Formula
   desc "Tool for securing CI/CD workflows with version pinning"
   homepage "https://github.com/sethvargo/ratchet"
-  url "https://github.com/sethvargo/ratchet/archive/refs/tags/v0.9.0.tar.gz"
-  sha256 "e6d418075866a0c1040ef36a009cb4f8d2ed0edf6f4b5cb0041c3d076c6a349a"
+  url "https://github.com/sethvargo/ratchet.git",
+    tag:      "v0.9.0",
+    revision: "8f1cce74d9a2fe51ff550a51b1a175d913981b7a"
```

Comment:
I'm not sure the implications of doing this? Will exiting bottles rebuild and use the ldflags?

Reply:
Newer installs will use the ldflags added here. Though, we try to avoid git checkouts when possible; they're slower and bad for caching.

Reply:
I couldn't figure out a way to get the git revision of the tarball without the checkout. Is there a helper function for that? If not, would you be open to the "commit" being something like "homebrew"?

Reply:
> I couldn't figure out a way to get the git revision of the tarball without the checkout. Is there a helper function for that?

Yes, I believe that there is no way to recover the Git revision without the checkout.

> If not, would you be open to the "commit" being something like "homebrew"?

That's perfectly fine, and what we do for some formulae that build with Go. If you're happy with that, then we're happy.

Reply:
Done!